devel/onetbb/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        oneapi-src oneTBB 2021.5.0 v
+name                onetbb
+revision            0
+categories          devel parallel
+platforms           darwin
+supported_archs     i386 x86_64
```

Comment:
this doesn't support `arm64` and or any of the other architectures that the `tbb` port builds for? Do we need to specify this at all or can we just use the default set?

Reply:
I didn't add `arm64` because I have no way of testing it. It seems there is [no official support](https://github.com/oneapi-src/oneTBB/issues/539) for ARM.

Reply:
so how about then not specifying the `supported_arch` and just see whether it builds on the buildbots and/or fine-tune this upon user reports?

Reply:
Hmm, alright. I didn't know not specifying `supported_archs` was an option tbh.